### ISSUE TITLE
Strange logs printed on require

### ISSUE THREAD

**Issue by hlolli** [akoidan/lines-logger#4]
I was testing this package, and finally I have the lines showing up correctly, definitely important for my flow.
I notice in current last version there's some "hello world" statement as well as strange log statements appearing when importing/requiring.

While this is nodejs, the same thing is happening in the browser:

```node
> const x = require("lines-logger/lib/browser");
[ { a: { b: 3 } } ]
undefined
```

**Comment by akoidan** [comment]
@hlolli could you please provide a print screen, I can't reproduce this issue
![image](https://user-images.githubusercontent.com/1336703/104667684-50e7db00-56df-11eb-9ada-939b510e6bd1.png)

**Comment by hlolli** [comment]
Here's a reproduceable example https://codepen.io/hlolli/pen/ZEpmPdz

**Comment by akoidan** [comment]
Fixed with version `2.1.2`
```html
<!doctype html>
<html lang="en">
<head>
  <script src="https://unpkg.com/lines-logger@2.1.2/lib/browser.js"></script>
  <script module>
      var LoggerFactory  = linesLogger.LoggerFactory;
      var loggerFactory = new LoggerFactory();
      var logger = loggerFactory.getLogger('tag');
      logger.log('asd')();
  </script>
</head>
<body>

</body>
</html>
```

**Comment by hlolli** [comment]
thanks!
